test/integration/typescript/next-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="next" />
+/// <reference types="next/types/global" />
```

Comment:
```suggestion
```

I think we shouldn't commit this file to make sure it's properly being added while testing, what do you think?